src/lib/api/UrlParamValidator.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of React-SearchKit.
+ * Copyright (C) 2020 CERN.
+ *
+ * React-SearchKit is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import _isArray from 'lodash/isArray';
+import _isNumber from 'lodash/isNumber';
+
+/** Default implementation for a param validator class */
+export class UrlParamValidator {
+  /**
+   * Return true if the param value is valid, false otherwise
+   * @param {object} urlHandler the url handler
+   * @param {string} key the parameter key
+   * @param {string} value the parameter value
+   */
+  isValid = (urlHandler, key, value) => {
+    switch (key) {
+      case 'queryString': case 'sortBy':
```

Comment:
funny formatting... I couldn't see the `sortBy`!

Reply:
Thanks, will add a line-break.